terraform/cloudfront/cloudfront_distribution/distribution.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_route53_record" "cdn_domain" {
+  count = "${length(var.aliases)}"
+
+  zone_id = "${var.system_dns_zone_id}"
+  name    = "${element(var.aliases, count.index)}."
+  type    = "CNAME"
+  ttl     = "60"
+  records = ["${aws_cloudfront_distribution.cdn_instance.domain_name}"]
+}
+
+resource "aws_cloudfront_distribution" "cdn_instance" {
+  origin {
+    domain_name = "${var.origin}"
+    origin_id   = "${var.origin}"
+
+    custom_origin_config {
+      origin_protocol_policy = "https-only"
+      http_port              = 80
+      https_port             = 443
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
```

Comment:
Not a blocker, but out of curiosity, I wonder if we need to support TLSv1 and TLSv1.1

There was some discussion about it recently, do you know the outcome @bleach @davbo @davidillsley ?

Sidenote: our ELBs do because the default AWS policies do: https://github.com/alphagov/paas-cf/blob/3e3ba8b18d2c9ccb6928713ad4e9e0859b0e92e5/terraform/globals.tf#L125